hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ConsensusConfig.java

```diff
@@ -24,7 +24,7 @@
 public record ConsensusConfig(
         @ConfigProperty(value = "message.maxBytesAllowed", defaultValue = "1024") @NetworkProperty
                 int messageMaxBytesAllowed,
-        @ConfigProperty(value = "message.maxPrecedingRecords", defaultValue = "950") @NetworkProperty
+        @ConfigProperty(value = "message.maxPrecedingRecords", defaultValue = "3") @NetworkProperty
```

Comment:
Won't this throw an exception for any genesis boot? (Since system accts have to be created)

Reply:
That is the correct value in mono-service.  I have added a condition `PrecedingTransactionCategory` enum to differentiate genesis case in the same PR